x-pack/plugins/infra/server/lib/domains/log_entries_domain/log_entries_domain.ts

```diff
@@ -4,6 +4,8 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
+import { sortBy } from 'lodash';
+import { InfraLogItem } from 'x-pack/plugins/infra/public/graphql/types';
```

Comment:
How about importing from the server-side code instead?

```suggestion
import { InfraLogItem } from '../../graphql/types';
```